src/main/java/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,76 @@
+package ladder.domain;
+
+import java.util.*;
+
+/**
+ * 사다리를 그려주는 클래스
+ * <br> 가로, 세로를 넣어준다.
+ * <br> Ladder ladder = new Ladder(5,4)
+ * <br> ladder.
+ *
+ * @author heebg, hyojaekim
+ * @version 1.0 2019-05-16
+ */
+public class Ladder {
+    private List<LadderLine> ladder;
+    private List<Player> players;
+
+    /**
+     * 생성자
+     *
+     * @param players
+     * @param depth
+     */
+    public Ladder(List<Player> players, int depth) {
+        this.ladder = getLadderLines(players.size(), depth);
+        this.players = players;
+        moveResultPosition(depth);
+    }
+
+    private List<LadderLine> getLadderLines(int playerSize, int depth) {
+        List<LadderLine> ladder = new LinkedList<>();
+        for (int i = 0; i < depth; i++) {
+            ladder.add(new LadderLine(playerSize));
+        }
+        return ladder;
+    }
+
+    private void moveResultPosition(int depth) {
+        for (int i = 0; i < depth; i++) {
+            moveOneLinePosition(i);
+        }
+    }
+
+    private void moveOneLinePosition(int depth) {
+        for (int i = 0; i < players.size(); i++) {
```

Comment:
ide에서 더 간결한 코드를 추천해주네요
```suggestion
        for (Player player : players) {
            ladder.get(depth).movePlayerPosition(player, player.getPosition());
        }
```

Reply:
for문을 만들 때 for each를 사용할 수 없는지 먼저 생각하고 만들어야겠다고 생각했습니다.. 감사합니다!